deploy/static/cert-manager-openshift-routes.yaml

```diff
@@ -1,130 +1,144 @@
+      verbs:
+        - get
+        - list
+        - watch
+    - apiGroups:
+      - ""
+      resources:
+      - events
+      verbs:
+      - create
+      - patch
+    - apiGroups:
+      - coordination.k8s.io
+      resources:
+      - leases
+      verbs:
+      - create
+      - get
+      - list
+      - update
+
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: cert-manager-openshift-routes
+      namespace: ${CERT_MANAGER_NAMESPACE}
+    automountServiceAccountToken: false
+
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: cert-manager-openshift-routes
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cert-manager-openshift-routes
+    subjects:
+    - kind: ServiceAccount
+      name: cert-manager-openshift-routes
+      namespace: ${CERT_MANAGER_NAMESPACE}
+
+  - apiVersion: apps/v1
+    kind: Deployment
     metadata:
+      name: cert-manager-openshift-routes
+      namespace: ${CERT_MANAGER_NAMESPACE}
       labels:
         app.kubernetes.io/name: cert-manager-openshift-routes
         app.kubernetes.io/version: "$RELEASED_VERSION"
         app.kubernetes.io/component: controller
-        app.kubernetes.io/part-of: cert-manager
+        app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
     spec:
-      serviceAccountName: cert-manager-openshift-routes
-      automountServiceAccountToken: true
-      containers:
-        - name: cert-manager-openshift-routes
-          image: "ghcr.io/cert-manager/cert-manager-openshift-routes:$RELEASED_VERSION"
-          args:
-          - -v=5
-          ports:
-          - containerPort: 6060
-            name: readiness
-            protocol: TCP
-          - containerPort: 9042
-            name: metrics
-            protocol: TCP
-          readinessProbe:
-            httpGet:
-              port: readiness
-              path: "/readyz"
-            initialDelaySeconds: 3
-            periodSeconds: 5
-            timeoutSeconds: 3
+      replicas: 1
+      selector:
+        matchLabels:
+          app.kubernetes.io/name: cert-manager-openshift-routes
+          app.kubernetes.io/version: "$RELEASED_VERSION"
+          app.kubernetes.io/component: controller
+          app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: cert-manager-openshift-routes
+            app.kubernetes.io/version: "$RELEASED_VERSION"
+            app.kubernetes.io/component: controller
+            app.kubernetes.io/part-of: ${CERT_MANAGER_NAMESPACE}
+        spec:
+          serviceAccountName: cert-manager-openshift-routes
+          automountServiceAccountToken: true
+          containers:
+            - name: cert-manager-openshift-routes
+              image: "ghcr.io/cert-manager/cert-manager-openshift-routes:$RELEASED_VERSION"
+              args:
+              - -v=5
```

Comment:
Just a heads up: you also need to set `--leader-election-namespace=${CERT_MANAGER_NAMESPACE}`, otherwise the controller will try to create resources for the leader election in the `cert-manager` namespace:

```
cert-manager-openshift-routes -h
cert-manager support for openshift routes

Usage:
  cert-manager-openshift-routes [flags]

App flags:

      --enable-leader-election             Whether to enable leader election on the controller. (default true)
      --leader-election-namespace string   Namespace to create leader election resources in. (default "cert-manager")
  -v, --log-level string                   Log level (1-5). (default "1")
      --metrics-port int                   Port to expose Prometheus metrics on 0.0.0.0 on path '/metrics'. (default 9402)
      --readiness-probe-path string        HTTP path to expose the readiness probe server. (default "/readyz")
      --readiness-probe-port int           Port to expose the readiness probe. (default 6060)

```